release/stable/centos7/docker/Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG fromTag=7
 FROM centos:${fromTag}
 
 ARG VCS_REF="none"
-ARG PS_VERSION=6.0.4
+ARG PS_VERSION=6.1.0
```

Comment:
some other dockerfiles (e.g. Ubuntu's, servercore) also have 6.0.4 as the default; probably also needs to be updated.

Reply:
We can update these separately.